nncf/common/quantization/quantizer_propagation/structs.py

```diff
@@ -9,6 +9,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import annotations
```

Comment:
Is it still required?

Reply:
Used to avoid " in annotations https://github.com/openvinotoolkit/nncf/pull/3192/files/f1acda47785191e736c597d07b8606ffbf755f37#diff-ad8299d77065a9fbb34eb2d6ad4967a419cf52ce5603988dc6b63f1616ea0d88L102-R106